docs/questions/sharing/visualizing-results.md

```diff
@@ -244,7 +244,7 @@ There's quite a bit you can do in this tab, but the options available will depen
 - **Change the style of your lines** for Line and Area charts, and choose whether to display dots on the lines.
 - **Specify how to handle missing values**. Use the "Replace missing values with…" setting to change how your chart deals with missing values. You can use linear interpolation, or display those points as zero or as nothing.
 - **Add a goal line**. Goal lines can be used in conjunction with [alerts](./alerts.md) to send an email or a Slack message when your metric cross this line.
-- **Add a trend line**. If you're looking at a time series chart, you can turn on a trend line to show where things are heading. Trend lines won't work if you have more than one time field grouping.
+- **Add a trend line**. If you're looking at a time series chart, you can turn on a trend line to show where things are heading. To add a trend line, you must choose exactly one time field grouping.
```

Comment:
This phrasing is more precise, but you should follow with an example of a single time field grouping, as "time field grouping" is an abstract phrase for people to parse.